client/my-sites/posts/main.jsx

```diff
@@ -127,30 +123,41 @@ const PostsMain = React.createClass( {
 
 	setWarning( selectedSite ) {
 		if ( selectedSite && selectedSite.jetpack && ! selectedSite.hasMinimumJetpackVersion ) {
-			notices.warning(
-				this.translate( 'Jetpack %(version)s is required to take full advantage of all post editing features.', { args: { version: config( 'jetpack_min_version' ) } } ),
-				{ button: this.translate( 'Update now' ), href: selectedSite.options.admin_url + 'plugins.php?plugin_status=upgrade' }
+			this.props.warningNotice(
```

Comment:
This seems like it would be prone to having duplicate notices shown. Would probably be a good idea to specify an `id` from [`createNotice`](https://github.com/Automattic/wp-calypso/blob/6a2d59128ef738f95ddfd32c68e50a05152ddd6a/client/state/notices/actions.js#L21) to ensure a maximum of one notice with this message. Not relevant to the changes here and can certainly be made separately, but figured I'd note this observation.